documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -37,7 +37,7 @@ Set the `disable_commandline_tags` command to `1` to stop `start.elf` from filli
 
 === `kernel`
 
-`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. On the Pi4, it is `kernel7l.img`.
+`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Raspberry Pi 1, Zero, Zero W and Raspberry Pi Compute Module is `kernel.img`, and on the Raspberry Pi 2, 3, 3+, Zero 2 W and Raspberry Pi Compute Module 3 and 3+ it is `kernel7.img`. On the Raspberry Pi 4, 400 and Raspberry Pi Compute Module 4 it is `kernel7l.img`.
```

Comment:
ping @helenlynn who can probably explain the rules better than I can.

Reply:
@lurch is correct. It's a just a matter of grammar, and I probably can't explain it as well as I can apply it, I'm afraid. In each case we need two separate lists: one list of "classic" and Zero line Raspberry Pi boards, and one list of Compute Modules, each with list items separated by commas. Correct grammar requires that we have two lists because in the names of "classic" and Zero line boards, "Raspberry Pi" is technically a noun phrase, whereas in the names of Compute Modules, "Raspberry Pi" is technically an adjectival phrase. Correct grammar also requires that the final item of such a list be separated from the penultimate item by "and" instead of, or as well as, a comma.

Reply:
@helenlynn 
I understand it. I will read similar sample in the technical document for more information. Thank you for your information.